sparkles/tests/test_checks.py

```diff
@@ -321,19 +321,47 @@ def test_guide_count_or():
         {'text': 'OR count of guide stars 3.65 < 4.0', 'category': 'critical'},
         {'text': 'OR with 4 guides but 5 were requested', 'category': 'caution'}]
 
-    # Set a scenario with guide_count in the 3.5 to 4.0 range and *but* with
-    # a creep away (maneuver angle <= 5), and confirm that is just a warning
-    # (not critical).
+
+def test_dither_low_guide_count():
```

Comment:
Make each of these individual tests into a distinct test function with appropriate docstring.